txscript/sign_test.go

```diff
@@ -670,9 +670,9 @@ func TestSignTxOutput(t *testing.T) {
 
 				keyDB, _, _, err := secp256k1.GenerateKey(rand.Reader)
 				key, pk := secp256k1.PrivKeyFromBytes(keyDB)
-				pkBytes := pk.SerializeUncompressed()
-
-				address, err := dcrutil.NewAddressSecpPubKey(pkBytes,
+				// TODO: this test might have to be removed since consensus rules
```

Comment:
These tests shouldn't be removed.  Instead, the comment should call out that they are required to be compressed currently due to the workaround in `ExtractPkScriptAddrs`.

Ultimately, the consensus rules need to be updated so they do not call into a standardness function.  Standard rules are intended to be a subset of consensus and invoking them is a mistake.